Modules/_datetimemodule.c

```diff
@@ -1911,6 +1960,7 @@ wrap_strftime(PyObject *object, PyObject *format, PyObject *timetuple,
         i++;
         /* A % has been seen and ch is the character after it. */
         PyObject *replacement = NULL;
+
```

Comment:
It's better to avoid unnecessary whitespace changes, as they can make future backports more difficult.